src/lib/log/src/lib.rs

```diff
@@ -132,6 +132,7 @@ pub fn print(args: fmt::Arguments) {
     LOGGER.lock().as_mut().unwrap().write_fmt(args).unwrap();
     #[cfg(not(feature = "mutex"))]
     unsafe {
+        // if let Some(l) = core::ptr::addr_of_mut!(LOGGER) { // not working
```

Comment:
Help wanted: any idea how to properly write this? This here is what the Rust toolchain suggests when compiling, but it errors.